dali/pipeline/data/buffer.h

```diff
@@ -123,6 +123,9 @@ class Buffer {
    * the non-const version of the method, or calling 'set_type'.
    */
   inline void* raw_mutable_data() {
+    // Empty tensor
+    if (size_ == 0 && data_.get() == nullptr)
```

Comment:
just wondering. Isn't one of those conditions enough to know that it's empy?

Reply:
You're right, done

Reply:
It seems to be not done yet.